tests/test_utils.py

```diff
@@ -55,6 +56,19 @@ def test_write_labels(tmpdir):
     assert(label == utils.read_labels(file_name=file_path))
 
 
+def test_get_ImageDataGenerator():
+    img_name = next(f for f in os.listdir('data/') if f.endswith('.png'))
+    img = utils.load_image(img_name)
+    data_gen = utils.get_ImageDataGenerator([img_name], img.shape)
+    assert(isinstance(data_gen, ImageDataGenerator))
+    assert data_gen.rescale == 1. / 255
```

Comment:
No need to test this, we should test that the rescaling happens, but not the property itself.